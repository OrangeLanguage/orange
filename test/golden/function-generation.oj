def const(x, y) x