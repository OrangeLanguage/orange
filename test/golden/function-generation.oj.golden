function _() {
  const const = function _(_handle, x, y) {
    x = x();
    y = y();
    return x;
  };
  return const;
}()